db/migrate/20170818020825_add_tsheets_user_name_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddTsheetsUserNameToUsers < ActiveRecord::Migration
+  def change
```

Comment:
I'm a bit worried about this change to users table. User is our GOD model, all the GOD models get polluted very fast -- and creating 3 columns just like that seems a bit redundant. 

Can I recommend to create a tsheet:jsonb column? Then you can store all the profile data from tsheet service in there as json, you can index individual fields (let's say tsheets_user_id). There are also other data types -- json, hstore that you may use here. But it seems that jsonb is better fit for this use case.

There is a good sum up on this here:
https://stackoverflow.com/questions/22654170/explanation-of-jsonb-introduced-by-postgresql